src/app/_hooks/useGetPathQuery.ts

```diff
@@ -0,0 +1,14 @@
+import { useQuery } from '@tanstack/react-query';
+import { getTaskPath } from '../_apis/tasks';
+
+const useGetPathQuery = (taskId: number, enabled: boolean) => {
+  return useQuery({
+    queryKey: [taskId],
```

Comment:
```suggestion
    queryKey: ['getPath', taskId],
```
이런 식으로 문자열 key가 있으면 devTools에서 쿼리 수행될 때 어떤 쿼리를 호출한건지 알기 쉬울 것 같아요!